manifests/infrastructure-roles.yaml

```diff
@@ -11,7 +11,8 @@ data:
   # testuser
   user2: dGVzdHVzZXI=
   # foobar
-  password2: Zm9vYmFy
+  password2: c2VjcmV0
+  batman: anVzdGljZQ==
```

Comment:
Please update the docs to mention (a) what infra roles are for and (b) that their passwords are still distributed as k8s secrets.